libs/auth-nest-tools/src/lib/current-auth.decorator.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  createParamDecorator,
+  ExecutionContext,
+  UnauthorizedException,
+} from '@nestjs/common'
+import { Auth } from './auth'
+import { getRequest } from './getRequest'
+
+export const getCurrentAuth = (context: ExecutionContext): Auth => {
+  const request = getRequest(context)
+
+  const auth = request.auth
+  if (!auth) {
+    console.warn(
+      'No authentication found. Did you forget to add IdsAuthGuard or IdsUserGuard?',
+    )
```

Comment:
Is logging to console intentional here or should this maybe use the logger?

Reply:
Good catch!